lib/routes/account.js

```diff
@@ -2459,7 +2459,8 @@ module.exports = (
             .then(
               function () {
                 // Notify the devices that the account has changed.
-                push.notifyPasswordReset(accountResetToken.uid, devicesToNotify)
+                var uid = Buffer(accountResetToken.uid, 'hex')
```

Comment:
Are you sure that `accountResetToken.uid` is a hex string here?  The `uid` field of `Token` objects is usually a buffer IIRC.